helix-term/src/commands.rs

```diff
@@ -2641,6 +2641,56 @@ pub mod cmd {
         Ok(())
     }
 
+    fn sort(
+        cx: &mut compositor::Context,
+        args: &[Cow<str>],
+        _event: PromptEvent,
+    ) -> anyhow::Result<()> {
+        sort_impl(cx, args, false)
+    }
+
+    fn sort_reverse(
+        cx: &mut compositor::Context,
+        args: &[Cow<str>],
+        _event: PromptEvent,
+    ) -> anyhow::Result<()> {
+        sort_impl(cx, args, true)
+    }
+
+    fn sort_impl(
+        cx: &mut compositor::Context,
+        _args: &[Cow<str>],
```

Comment:
Keeping this here as I assume there could be args for case-insensitivity, number sorting, etc. Maybe it might even be worth it to extract the whole implementation into some `Sorter`.

Reply:
Not a maintainer, but I don't see that being necessary unless there were many commands sharing this same function. A single option seems fine to leave as a bool.